src/content/docs/tutorial-external-services/respond-external.mdx

```diff
@@ -0,0 +1,147 @@
+
+This tutorial series shows you how to identify problematic API interactions and how to resolve them with the New Relic platform. This doc covers:
+
+* Integrating New Relic with your application to send data to monitor
+* Identifying problematic APIs through the external services UI
+
+## Integrate New Relic [#integrate]
+
+Before you can monitor or resolve anything, you need to install an agent to collect the data you'll use. 
+
+<Steps>
+    <Step>
+
+### Install an APM agent
+To monitor your application's performance, you'll use an agent created specifically for your app's language. Clicking a logo sends you to New Relic platform where you'll be guided through installing and configuring the agent.
+
+<TechTileGrid>
+  
+  <TechTile
+    name="Go agent"
+    icon="logo-go"
+    to="https://one.newrelic.com/nr1-core?state=985d4005-ba90-a8c7-1da1-2af34539b03b"
+  />
+
+  <TechTile
+    name="Java agent"
+    icon="logo-java"
+    to="https://one.newrelic.com/nr1-core?state=80d18bcb-4919-1fcb-2b77-9406838eb916"
+  />
+
+  <TechTile
+    name=".NET agent"
+    icon="logo-dotnet"
+    to="https://one.newrelic.com/nr1-core?state=30e93090-6dfa-6b70-8e75-472f54414355"
+  />
+
+  <TechTile
+    name="Node.js agent"
+    icon="logo-nodejs"
+    to="https://one.newrelic.com/marketplace/install-data-source?state=be2e62fa-cc3b-c428-27c4-8d662c9e80a1"
+  />
+
+  <TechTile
+    name="PHP agent"
+    icon="logo-php"
+    to="https://one.newrelic.com/nr1-core?state=aa633b41-72d4-009c-3abf-55dcf64894fe"
+  />
+
+  <TechTile
+    name="Python agent"
+    icon="logo-python"
+    to="https://one.newrelic.com/nr1-core?state=20fda75b-58fb-a92a-f9e1-7b052035c6e8"
+  />
+
+  <TechTile
+    name="Ruby agent"
+    icon="logo-ruby"
+    to="https://one.newrelic.com/nr1-core?state=d69143ab-605c-579b-25bf-cc6e5fee5b80"
+  />
+</TechTileGrid>        
+
+Once you've installed an agent, go to **[one.newrelic.com](https://one.newrelic.com/nr1-core?filters=(domain%3D'APM'ANDtype%3D'APPLICATION'))** and select your app. If you don't see much data just yet, step away for a while and let the agent gather real-time data as your application runs.
+    </Step>
+    <Step>
+
+### Additional setup
+
+Your specific configuration may require additional setup. Take a look at our [external service setup doc](/docs/apm/apm-ui-pages/monitoring/external-services/external-services-setup/) for more information.
+
+Once you've completed any additional steps from the doc above, go to **[one.newrelic.com](https://one.newrelic.com/nr1-core?filters=(domain%3D'APM'ANDtype%3D'APPLICATION'))** > select your app > *External services**. You should see data about your APIs and external services flow in.
+    </Step>
+</Steps>
+
+## Identify problematic APIs 
+
+You now have the ability to track how your application interacts with APIs and other external services. Let's use that info to identify which APIs or external services are causing issues:
+
+<Steps>
+    <Step>
+### Think about the root cause 
+
```

Comment:
This whole doc is assuming that a user is coming to New Relic because they're getting some sort of feedback that their site/application isn't working. Is it possible to identify ways New Relic might let them know something is wrong? Or is the premise basically: my site isn't working, let me check API calls? Also-- how does this play with alerts/APM? I feel like there are opportunities to integrate and weave different areas together?

Reply:
This is a solid point. The audience for this is basically "I'm 90% sure I have an API issue, let me learn how to use New Relic to troubleshoot that one specific slice of my system."

Our _root causes_ tutorial is a more general "my app is slow" let me use APM, alerts, SLOs, etc. to take a wider look at my system.

This tutorial is a thin slice on purpose